acct/check

```diff
@@ -132,11 +135,31 @@ if [ -f "$vhost_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$v
 fi
 
 # check apphosting config
-if [ -f "$vhost_app_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
+if [ -f "$vhost_app_file" ] && vhost_app=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
   echo
   echo "Virtual apphosts:"
   echo "-------------------------------------------------------------------------------"
-  echo "$vhost"
+  echo "$vhost_app"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check virtual mail config
+if [ -f "$vhost_mail_file" ] && vhost_mail=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_mail_file"); then
+  echo
+  echo "Virtual mail:"
+  echo "-------------------------------------------------------------------------------"
+  echo "$vhost_mail"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check dns entries
+dns=$("$(dirname "$(readlink -f -- "$0")")"/check-dns "$user")
```

Comment:
@kpengboy looks good?

Reply:
Looks good, would you be able to fix update-symlinks too?

Reply:
done